### PR TITLE
One sentence per line

### PR DESCRIPTION
**Pull request by aollier**
I made changes to get only one sentence per line and one line per sentence (with rare exceptions). I also changed the prefaces to get this rule applied to be consistent with the book.

I also corrected a typo that I saw while re-reading.

### CODE REVIEW

File: book/07-git-tools/sections/submodules.asc [hunk 1]
```diff
@@ -682,7 +682,9 @@ CONFLICT (submodule): Merge conflict in DbConnector
 Automatic merge failed; fix conflicts and then commit the result.
 ----
 
-What it's suggesting that you do is to update the index like you had run `git add`, which clears the conflict, then commit. You probably shouldn't do this though. You can just as easily go into the submodule directory, see what the difference is, fast-forward to this commit, test it properly, and then commit it.
+What it's suggesting that you do is to update the index like you had run `git add`, which clears the conflict, then commit.
```
Comment (HonkingGoose) [inline book/07-git-tools/sections/submodules.asc]: This line is a bit clunky to read. But I myself have no clear improvement. Maybe @ben can offer help with a light rewrite of this line?
>What it's suggesting that you do is to update the index like you had run `git add`, which clears the conflict, then commit.

**Comment by aollier** [comment]
@ben Is there a problem with this PR ?

File: book/07-git-tools/sections/submodules.asc (hunk 1 above)
Comment (ben) [reply book/07-git-tools/sections/submodules.asc]: Sorry for the delay. How about:

> The suggested command Git is providing will update the index as though you had run `git add` (which clears the conflict), then commit.

Comment (HonkingGoose) [reply book/07-git-tools/sections/submodules.asc]: Hi @ben, the new line is a lot better.

---

Hi @aollier do you want to fix the line? Or would you like me to make a new pull-request?

Comment (aollier) [reply book/07-git-tools/sections/submodules.asc]: Ok I do it right now.

File: book/06-github/sections/2-contributing.asc [hunk 2]
```diff
@@ -457,7 +457,7 @@ When rendered, it would look something like <<_md_emoji>>.
 .Heavy emoji commenting.
 image::images/markdown-07-emoji.png[Emoji]
 
-Not that this is incredibly useful, but it does add an element of fun and emotion to a medium that is otherwise hard to convey emotion in.
```
Comment (ScumCoder) [inline book/06-github/sections/2-contributing.asc]: I am pretty sure this version was correct.

Comment (ben) [reply book/06-github/sections/2-contributing.asc]: Ha! You’re completely right. Want to submit a pr that fixes it?

Comment (ScumCoder) [reply book/06-github/sections/2-contributing.asc]: Done (#1268).

**Review by felixthecat420 — APPROVED** [review]

**Pull request merged**